_summaries/zeiler2012adadelta_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{zeiler2012adadelta}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author:  sritank
+score:  10
 ---
 
 TODO: Summarize the paper:
+
 * What is the core idea?
+- Adaptively change the learning rate in a way that it reduces oscillations i.e. slows down close to minimas, and is large otherwise.
+- Change step size based on the windowed accumulated gradient and also accumulated step sizes, which only require 1st order computations.
+- Also scale the step size by the accumulated previous step sizes, giving it the correct units as if it were a 2nd order adaptive method
+
 * How is it realized (technically)?
+- Scaling gradient helps ensure we take an equal step in all directions
+- Builds on ADAGRAD which scales the learning rate based on the accumulated sum of previous gradients. 
+
+![ADAGRAD_step](ADAGRAD.png)
+- By choosing an exponential decaying average for accumulating the gradient they solved the 0 step size problem of ADAGRAD
+
+![ADADELTA_avg](exp_decay_avg.png)
+
+![ADADELTA_denom](exponential_decay_average.png)
+where epsilon is used to condition the denominator for numerical purposes.
+- By assuming locally smooth curvature, they scaled up the step size using the exponentially decaying average of previous step sizes.
+
+![ADADELTA_step](ADADELTA_step.png)
 * How well does the paper perform?
-* What interesting variants are explored?
+- numerator RMS term lags behind the denominator by 1 time step, making it robust to large sudden gradients. Denominator increases slowing down the progress before numerator can blow up.
+- The method makes approximations on Hessian and local curvature, giving it second order characteristics while still costing only one gradient computation per iteration.
+- Training method is used on DNNs with sigmoid and ReLU units. The DNNs were trained for MNIST classification and speech recognition.
```

Comment:
Was the training only on DNNs with sigmoid and ReLU by design? Would you expect ADADELTAto work / not work on DNN's with different non-linear activations?

Reply:
I believe the method is independent of the activation function. As long as we can calculate the gradient at each time step, the adaptive learning rate can be calculated at any point in the training. 
In their experiments they only seem to have tried sigmoid and ReLU